services/opencollective/opencollective-backers.service.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+const Joi = require('joi')
+const BaseJsonService = require('../base-json')
+
+// https://developer.opencollective.com/#/api/collectives?id=get-members
+const membersArraySchema = Joi.array().items(
+  Joi.object().keys({
+    MemberId: Joi.number().required(),
+    type: 'USER',
+    role: Joi.string().required(),
+  })
+)
+
+module.exports = class OpencollectiveBackers extends BaseJsonService {
+  static get category() {
+    return 'funding'
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'open collective backers',
+        namedParams: { collective: 'shields' },
+        staticExample: this.render('shields', 25),
+        keywords: ['opencollective'],
```

Comment:
I don't think you'll need these since the service name/title/route contain this already

Reply:
In this case, the badges will not be found by the query "opencollective".
In my opinion, the keyword is necessary.

Reply:
the service is called Open Collective though is it not? It's not a huge deal either way, but in other services where there is a space in the service name I don't think we typically add keywords without spacing/characters

Reply:
F-droid and vscode-marketplace are two that come to mind. I think it would be good if we matched the base URL. It might be nice to tackle that in a more automated way sometime, though until then I think listing them again is a good solution.

Reply:
To expound on that a bit: I think the reason `opencollective` should match is that it is part of the URL.

Reply:
> To expound on that a bit: I think the reason `opencollective` should match is that it is part of the URL.

Agreed on that and on a more automated way as I think the url part should match on search for all services. In some of those other services `Eclipse Marketplace` for example the url part used dashes for the spaces in lieu of merging the words (`eclipse-marketplace` vs `eclipsemarketplace`)

Another area where'd I'd vote for having consistency across services as much as possible (def. out of scope here though). As a user I shouldn't have to try to figure out whether I need dashes vs. combining the words, etc. IMO